sec/insecure/insecure.go

```diff
@@ -28,28 +29,36 @@ const ID = "/plaintext/2.0.0"
 // No authentication of the remote identity is performed.
 type Transport struct {
 	id  peer.ID
-	key ci.PrivKey
+	key crypto.PrivKey
```

Comment:
The reason we use a different import alias is to differentiate from the Go SDK crypto. I don't have strong opinions either way, just noting it.